game-service/src/main/liberty/config/server.xml

```diff
@@ -11,7 +11,12 @@
         <feature>mpRestClient-1.1</feature>
         <feature>mpOpenAPI-1.0</feature>
         <feature>websocket-1.1</feature>
+        <feature>mpMetrics-1.1</feature>
     </featureManager>
+    
+    <mpMetrics authentication="false"/>
+    
+    <logging traceSpecification="com.ibm.ws.microprofile.metrics*=all"/>
```

Comment:
this trace spec can be removed, I'm guessing you don't need it anymore and that it produces a lot of trace